src/pages/dmt5-trading-signals/_signal-steps.js

```diff
@@ -72,16 +72,25 @@ const content = {
                 ]}
             />
         ),
-        step_two_header: <Localize translate_text="2. Fill the broker field" />,
+        step_two_header: <Localize translate_text="2. Fill in your details" />,
         step_two_text: (
             <Localize
-                translate_text="Complete the form with your Deriv MT5 account credentials. For the Broker field, add <0>Deriv-Server</0> as your broker or <0>Deriv-Demo</0> if your signal is for demo accounts only."
-                components={[<strong key={0} />]}
+                translate_text="Complete the form with your Deriv MT5 account credentials. In the <0>Broker</0> field, enter your account server name:< /br><0>- Deriv-Demo</0> if your signal is for demo accounts only < /br> <0>-Deriv-Server</0>  or <0>Deriv-Server-02 </0> if your signal is for real accounts only. (You can find the account server name on your <1>Deriv MT5 dashboard</1>.)"
+                components={[
+                    <strong key={0} />,
+                    <LocalizedLinkText
+                        external="true"
+                        to="https://app.deriv.com/mt5"
+                        color="red"
+                        size={16}
+                        key={1}
+                    />,
```

Comment:
```suggestion
                    <LocalizedLinkText
                        key={1}
                        external="true"
                        to="https://app.deriv.com/mt5"
                        color="red"
                        size={16}
                    />,
```

small nit, best if the key is put as the first prop 🙏🏼

Reply:
Noted, thanks!